TESTS/netsocket/test_params.h

```diff
@@ -0,0 +1,29 @@
+#ifndef MBED_CONF_APP_ECHO_SERVER_ADDR
```

Comment:
Add license header on top of any new file

Reply:
Done